contracts/CO30_Matias_BlackSmokeRising.blud_cordelia/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,13 @@
+Terrain in the AO is covered with thick jungles, making traversal difficult outside of the river and paths.
+
+<font color='#e3d310'>EMPLOYER</font>
+Name: CIA (Central Intelligence Agency)
+
+<font color='#d81717'>ENEMY FORCES</font>
+Name: Local Militia
+Location: Several camps along the coast
```

Comment:
Location(s), and you are also giving a description here, not pointing out the camps themselves, alternatively you can also use the name of the Camps you have placed down as markers.